src/configCacheHandler.ts

```diff
@@ -15,6 +17,12 @@ const PRETTIER_CONFIG_FILES = [
     'prettier.config.js',
 ];
 
+const prettierConfigPath = getConfig().config;
```

Comment:
There may be multiple config files, one per workspace folder.
This is just reading one.

To do it correctly, we would need to set up file change hooks for every files. And remove them when needed. 

When a workspace folder is added, check the config and register a hook for it if needed
When a workspace folder is removed, remove the file watcher if it exists